second_sql_assignment.sql

```diff
@@ -0,0 +1,232 @@
+CREATE DATABASE TASK2; 
+
+USE TASK2;
+
+CREATE TABLE Employees(
+EmployeeID INT PRIMARY KEY,
+FirstName VARCHAR(50),
+LastName VARCHAR(50),
+Department VARCHAR(50),
+Salary DECIMAL(10,2),
+HireDate Date);
+
+INSERT INTO  Employees VALUES
+(1,'Gokul','Das','IT',70000,'2008-07-18'),
+(2,'Akhil','Philip','Finance',50000,'2010-06-20'),
+(3,'Sangeeth','Kumar','HR',45000,'2014-01-13'),
+(4,'Priya','Santhosh','Marketing',85000,'2005-04-11');
+
+
+-- CREATE A TABLE STUDENTS AND INSERT NAMES IN MALAYALAM.
+
+
+CREATE TABLE Students(
+StudentID INT PRIMARY KEY,
+StudentName VARCHAR(50) NOT NULL);
+
+
+ALTER TABLE Students
+ALTER COLUMN StudentName NVARCHAR(50);
+
+
+INSERT INTO Students (StudentID, StudentName) VALUES
+(1, N'ആനന്ദ്'),
+(2, N'സന്ദീപ്'),
+(3, N'വിജയ്കുമാർ');
+
+
+-- RETRIEVE ALL EMPLOYEES WHO WORK IN SALES, MARKETING OR IT DEPARTMENTS.
+
+
+SELECT * FROM  Employees
+WHERE Department IN ('Sales','Marketing','IT');
+
+-- FIND ALL EMPLOYEES WITH SALARIES RANGING FROM $50,000 TO $75,000 (INCLUSIVE)>
+
+SELECT * FROM  Employees
+WHERE Salary BETWEEN 50000 AND 75000;
+
+-- LIST ALL EMPLOYEES WHOSE LAST NAME STARTS WITH THE LETTER 'S'.
+
+SELECT * FROM Employees 
+WHERE LastName LIKE 'S%';
+
+-- DISPLAY ALL EMPLOYEES WITH EXACTLY 5 LETTERS IN THEIR FIRST NAME.
+
+SELECT * FROM Employees
+WHERE LEN(FirstName) = 5;
+
+-- FIND EMPLOYEES WHOSE LAST NAME STARTS WITH EITHER 'B','R', OR 'S'.
+
+
+SELECT * FROM Employees
+WHERE LEFT(LastName,1) IN ('B','R','S');
```

Comment:
Is there any other approach ? using Like operator

Reply:
SELECT * 
FROM Employees
WHERE LastName LIKE 'B%' or LastName LIKE 'R%' or LastName LIKE 'S%';